Formula/latexml.rb

```diff
@@ -0,0 +1,38 @@
+class Latexml < Formula
+  desc "LaTeX to XML/HTML/MathML Converter"
+  homepage "http://dlmf.nist.gov/LaTeXML"
+  url "https://github.com/brucemiller/LaTeXML/archive/v0.8.1.tar.gz"
+  sha256 "2ba1a580258ff5c7e3d9c2b40fd15cb4c92e388e5cd6b6127e8fbf9b1b9c63ce"
+
+  depends_on "cpanminus" => :recommended
```

Comment:
I dunno about making a dependency for `cpanminus` if it's just for the user to install `Image::Size` with, per the caveats. They can pick and install their own CPAN too, I think.